src/sinks/loki/config.rs

```diff
@@ -93,8 +93,8 @@ pub struct LokiConfig {
     ///
     /// Both keys and values are templateable, which enables you to attach dynamic labels to events.
     ///
-    /// Labels can be `*` or prefixes ending with `*` to allow the expansion of objects into
-    /// multiple labels, see [Label expansion][label_expansion] for more information.
+    /// Valid label keys include `*`,  and prefixes ending with `*`, to allow for the expansion of
```

Comment:
Oopise. my suggestion had an extra space before "and".

Reply:
Removed in [4c68791](https://github.com/vectordotdev/vector/pull/16591/commits/4c68791cc6cf8799607fd2a4564c323d13ca0d46)